### DISCUSSION TITLE
which theme is completely compatible with force blur?

### DISCUSSION THREAD

**Discussion by kc4ca** [taj-ny/kwin-effects-forceblur#100] [Q&A]
i installed the theme on nobara linux it works fine on some apps but terrible on some apps. i used the command on readme page to detect apps' window classes but i still can't get it working properly on brave browser, system settings and more. i also installed a semi transparent telegram theme but only the chat menu was blurred. the right click also seems to not work properly. also the top panel isn't blurred with force blur, i use a custom plasma style named macos sonoma. do i need to apply a compatible plasma theme?
![Screenshot_20240816_182312](https://github.com/user-attachments/assets/818c9420-d7a0-4943-a703-7714445ed7d6)
![Screenshot_20240816_182336](https://github.com/user-attachments/assets/eb055197-10f5-46be-9341-bb6e00f2787a)
![Screenshot_20240816_182437](https://github.com/user-attachments/assets/e7ce43dc-db37-48a3-b96f-2915705ba167)
![Screenshot_20240816_182610](https://github.com/user-attachments/assets/737e84f3-301c-4744-8f3e-2da9cfb5bdbd)
![Screenshot_20240816_182635](https://github.com/user-attachments/assets/50c22a9b-2284-474d-9fe1-ea929e48f5ce)

**Comment by taj-ny** [comment]
Blur will only be visible in areas of the window that are transparent.
https://github.com/taj-ny/kwin-effects-forceblur?tab=readme-ov-file#window-transparency


> the right click also seems to not work properly

I can't do anything about this, see #17. #89 is also related since you're using Kvantum.

> also the top panel isn't blurred with force blur

It is blurred though.

**Reply by kc4ca** [reply]
i applied the window rule to reduce opacity it works fine but i feel a well-made plasma theme could make this looking a lot better. thanks for your help

**Comment by this-username-system-sucks-ass** [comment]
I've had pretty good results from alphablack (plasma style, really configurable) and alpha (colors, tranparent/glassy, several different presets available), as well as klassy for my window decs/application style (bc it's really customizable, also has great presets including a mac-like one). I've also heard kvantum is good but in my experience it didn't do much for transparency.

my normal theme:

<img width="1920" height="1080" alt="Screenshot_20250717_144627" src="https://github.com/user-attachments/assets/9598e35b-4131-45ae-8fc5-02dd1b171a66" />

with the mac-like preset I mentioned:


<img width="1920" height="1080" alt="Screenshot_20250717_144830" src="https://github.com/user-attachments/assets/8cf6bfa3-cd52-4a53-b8b6-f441802a7068" />
you can't see it bc none of the windows are focused, but the traffic light colors on the left are just like mac and the extra stuff on the right is my accent color.
